TimeTable/Categories/WorkTime/WorkTimeViewModel.swift

```diff
@@ -24,7 +25,11 @@ protocol WorkTimeViewModelType: class {
     func viewDidLoad()
     func viewRequestedForNumberOfProjects() -> Int
     func viewRequestedForProjectTitle(atRow row: Int) -> String?
+    func viewRequestedForNumberOfTags() -> Int
+    func viewRequestedForTag(at index: IndexPath) -> ProjectTag?
     func viewSelectedProject(atRow row: Int)
+    func viewSelectedTag(at index: IndexPath)
+    func viewRequestIfTagIsSelected(at index: IndexPath) -> Bool
```

Comment:
`viewDidRequest...` or `viewRequested...` which is probably better for the moment, because every other function is named in this convention.